File: CS/ManageUsersOnLogon.Win/WinApplicationEx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Win;

namespace ManageUsersOnLogon.Win {
    public partial class ManageUsersOnLogonWindowsFormsApplication {
        public ManageUsersOnLogonWindowsFormsApplication() {
            InitializeComponent();
            DelayedViewItemsInitialization = true;
            EnsureShowViewStrategy();//Q438300
        }
        protected override void OnLoggedOn(LogonEventArgs args) {//Q477869
            base.OnLoggedOn(args);
            ShowViewStrategy = CreateShowViewStrategy() as WinShowViewStrategyBase;
        }
    }
}

File: VB/ManageUsersOnLogon.Win/WinApplicationEx.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Win

Namespace ManageUsersOnLogon.Win
	Partial Public Class ManageUsersOnLogonWindowsFormsApplication
		Public Sub New()
			InitializeComponent()
			DelayedViewItemsInitialization = True
			EnsureShowViewStrategy() 'Q438300
		End Sub
		Protected Overrides Sub OnLoggedOn(ByVal args As LogonEventArgs) 'Q477869
			MyBase.OnLoggedOn(args)
			ShowViewStrategy = TryCast(CreateShowViewStrategy(), WinShowViewStrategyBase)
		End Sub
	End Class
End Namespace
